testsuite/libffi.call/cls_dbls_struct.c

```diff
@@ -57,10 +57,10 @@ int main(int argc __UNUSED__, char** argv __UNUSED__)
 	CHECK(ffi_prep_closure_loc(pcl, &cif, closure_test_gn, NULL, code) == FFI_OK);
 
 	((void*(*)(Dbls))(code))(arg);
-	/* { dg-output "1.0 2.0\n" } */
+	/* { dg-output "1.0 2.0" } */
```

Comment:
This test fails on my DejaGnu board with these lines:

    /* { dg-output "1.0 2.0\n" } */
    /* { dg-output "1.0 2.0\n" } */

At the same time another tests in the test suite use another syntax like this and it works on my board and on x86:

    /* { dg-output "1.0 2.0" } */
    /* { dg-output "\n1.0 2.0" } */

I don't know why exactly the first variant does not work but the second one works everywhere and is used everywhere in the test suite.